### ISSUE TITLE
Make sure running works as intended with Grunt 1

### ISSUE THREAD

**Issue by kokarn** [kokarn/atom-grunt-runner#89]
There are some awesome improvements coming soon, we should make sure they work as intended.
http://gruntjs.com/blog/2016-02-11-grunt-1.0.0-rc1-released

The main part i'm worried about is using grunt without grunt cli but this in `package.json`
```json
"scripts": {
  "grunt": "grunt"
}
```

**Comment by ealves-pt** [comment]
Yes, we need to start preparing the code for that configuration method. I have marked this as an enhancement.

**Comment by kokarn** [comment]
Now that [Grunt 1.0.0 is released](http://gruntjs.com/blog/2016-04-04-grunt-1.0.0-released) this needs to be adressed. 
Confirmed not working right now.

The expected behaviour should probably be to check if `npm run grunt` is defined before running grunt. 
If it is, use that instead of just `grunt`.
Thoughts @ealves-pt ?
